classes/Gender.php

```diff
@@ -31,6 +31,10 @@
  */
 class GenderCore extends ObjectModel
 {
+    public const GENDER_MALE = '0';
+    public const GENDER_FEMALE = '1';
+    public const GENDER_NEUTRAL = '2';
```

Comment:
or, better: you add an other value "unknown", that becomes the default.

Reply:
- we keep the existing fixtures (Mr/Mme)
- when creating a new one --> “other” is set by default as the non-gendered option. "Other" replaces “neutral”
- “unknown” is unnecessary as it would essentially create a virtual double of “other” without any value since this gender field is tied to the fixtures.